dev/Dockerfile-dev

```diff
@@ -23,5 +23,6 @@ EXPOSE 9002
 EXPOSE 9003
 
 ENV OPENSLIDES_BACKEND_DEBUG 1
+ENV OS4_DEVELOPMENT 1
```

Comment:
Is this an env variable defined by the authlib? Please add an explaination to the README.

Reply:
We'll add this env variable for every service, since multiple service need the info "I'm in development mode" and we just use `OS4_DEVELOPMENT` for every service

Reply:
It's essentially the same as previously `OPENSLIDES_BACKEND_DEBUG`. We should drop the latter and just use `OS4_DEVELOPMENT` in the backend.